thorlcr/graph/thgraph.cpp

```diff
@@ -1395,6 +1395,9 @@ void CGraphBase::executeSubGraph(size32_t parentExtractSz, const byte *parentExt
     CriticalBlock b(executeCrit);
     if (job.queryPausing())
         return;
+    PerfTracer perf;
+    if (job.getOptBool("perftrace"))
```

Comment:
question: should this be an expert option (use getExpertOptBool)

Reply:
question: should this be restricted to run for parent subgraphs, not each child graph execution?
If so, should move inside the if (!queryOwner()) blocks.

Reply:
I don't know which is likely to be more useful...

I suspect you probably want to be able to select a particular graph (or subgraph) via hints or similar but that is left as an exercise for the future...

Reply:
I did wonder about expert option - it's more likely you'll want to set as a workunit option for a single run rather than a global option for a thor deployment, and I'm not sure if expert works there?

Reply:
right, yeah expert settings are only picked up at the component, or global level at the moment, not from workunit settings.

Reply:
> I don't know which is likely to be more useful...
> 
> I suspect you probably want to be able to select a particular graph (or subgraph) via hints or similar but that is left as an exercise for the future...

might be worth moving for now, because otherwise each child execution either has to have new filename output(?) or it will end up overwriting per child graph execution